### PR TITLE
Updated IridiumCore and dependent APIs

### PR DESCRIPTION
**Pull request by sh0inx**
mostly build script changes
- updated IridiumCore to 1.8.9
- updated IridiumSkyblockAPI to 4.0.8
- updated SuperiorSkyblockAPI to use new repo
- updated script to include LandsAPI because this wasn't here for some reason
- removed MassiveCore and MassiveCore Factions because this is no longer available to compile with - [see their github profile](https://github.com/MassiveCraft)
- removed MWDPlaceholders for the same reason

### CODE REVIEW

File: src/main/java/com/iridium/iridiumenchants/support/FactionsUUIDSupport.java
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumenchants/support/FactionsUUIDSupport.java:1]: Why was FactionsUUID support removed?

File: build.gradle.kts [hunk 1]
```diff
@@ -5,7 +5,6 @@ plugins {
 }
 
 group = "com.iridium"
-version = "4.1.5"
```
Comment (PeachesMLG) [inline build.gradle.kts:8]: Why was this removed?

File: src/main/java/com/iridium/iridiumenchants/support/FactionsUUIDSupport.java
Comment (sh0inx) [reply src/main/java/com/iridium/iridiumenchants/support/FactionsUUIDSupport.java:1]: I cant find a way to compile it - massivecraft seems to have taken their repos offline and abandoned their github

File: build.gradle.kts (hunk 1 above)
Comment (sh0inx) [reply build.gradle.kts:8]: great question
